draft-ietf-quic-recovery.md

```diff
@@ -414,6 +445,15 @@ acknowledged.  DetectLostPackets is called every time there is a new largest
 packet or if the loss detection alarm fires the previous largest acked packet is
 supplied.
 
+### Handshake Packets
+
+The receiver MUST not trust an unencryped ack for encrypted packets.  The receiver
```

Comment:
s/unencryped/unencrypted

More generally, use "protected" rather than "encrypted' please.  We have both confidentiality and integrity protection and both are relevant.